src/SamplesApp/UITests.Shared/Windows_UI_Input/PointersTests/Nested_Exit.xaml

```diff
@@ -0,0 +1,34 @@
+﻿<Page
+	x:Class="UITests.Windows_UI_Input.PointersTests.Nested_Exit"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:UITests.Windows_UI_Input.PointersTests"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	mc:Ignorable="d"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<Grid>
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition />
+			<ColumnDefinition />
+		</Grid.ColumnDefinitions>
+
+		<Grid.RowDefinitions>
+			<RowDefinition />
+			<RowDefinition />
+		</Grid.RowDefinitions>
+
+		<TextBlock Text="With intermediate non hit-testable object" />
+
+		<Border Background="DeepPink" Width="128" Height="128" PointerExited="Case1_OnPointerExited" x:Name="Case1_Pink">
+			<Border Background="{x:Null}" Width="160" Height="160" Margin="-32">
```

Comment:
What does this `x:Null` do to the Background?

Reply:
It will make the element non hit-testable, but without flagging children also as non hit-testable.